test/apis/realtime/image-classifier-resnet50/inferentia.md

```diff
@@ -0,0 +1,69 @@
+# Inferentia
+
+The Inferentia examples were inspired by [this tutorial](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-deploy/tutorials/k8s_rn50_demo.html).
```

Comment:
With respect to the entire `md` file: maybe we can add a comment explaining that this should naturally be hit from outside the cluster, not by `exec`ing into the pod and running these. What do you think?